### ISSUE TITLE
[hold] In backend, remove content pages from configurable routing

### ISSUE THREAD

**Issue by Platonn** [SAP/cloud-commerce-spartacus-storefront#663]
- contact
- help
- sale
- termsAndConditions

**Comment by tobi-or-not-tobi** [comment]
Blocked by #353

**Comment by Platonn** [comment]
Related to #846

**Comment by Xymmer** [comment]
postpone until... outcome of discussions for generic content pages (Kris)

**Comment by Xymmer** [comment]
closing as not needed
